### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Relating to https://github.com/itering/internal-security-audit/pull/85

### CODE REVIEW

File: runtime/darwinia/src/pallets/tx_pause.rs [hunk 1]
```diff
@@ -19,15 +19,27 @@
...
+				| b"Assets" | b"Vesting"
+				| b"Deposit" | b"AccountMigration"
+				| b"DarwiniaStaking"
+				| b"Ethereum" | b"EVM"
+				| b"MessageTransact"
+				| b"BridgePolkadotGrandpa"
+				| b"BridgePolkadotParachain"
+				| b"BridgeDarwiniaMessages"
+				| b"BridgeDarwiniaDispatch"
+				| b"DarwiniaFeeMarket"
```
Comment (boundless-forest) [inline runtime/darwinia/src/pallets/tx_pause.rs:42]: Would you like to expand the current `impl_maintenance_tests` to ensure this works as intended.

**Review by boundless-forest — APPROVED** [review]

**Comment by hackfisher** [comment]
@AurevoirXavier Are we still going to disable tx-pause in Darwinia runtime as it was concluded in the audit meeting?

**Comment by AurevoirXavier** [comment]
> @AurevoirXavier Are we still going to disable tx-pause in Darwinia runtime as it was concluded in the audit meeting?

I've changed from using a whitelist to a blacklist.

Most calls will no longer be able to pause.

What's your opinion?

**Comment by hackfisher** [comment]
> > @AurevoirXavier Are we still going to disable tx-pause in Darwinia runtime as it was concluded in the audit meeting?
> 
> I've changed from using a whitelist to a blacklist.
> 
> Most calls won't be able to pause.
> 
> What's your opinion?

I think we should disable it in Darwinia runtime as this pallet is not audited yet in the upstream.

**Pull request merged**